eng/common/templates/phases/publish-build-assets.yml

```diff
@@ -28,12 +28,15 @@ phases:
             SecretsFilter: 'MaestroAccessToken'
           condition: ${{ parameters.condition }}
           continueOnError: ${{ parameters.continueOnError }}
-        - script: eng\common\publishbuildassets.cmd
-            /p:ManifestsPath='$(Build.StagingDirectory)/Download/AssetManifests'
-            /p:BuildAssetRegistryToken=$(MaestroAccessToken)
-            /p:MaestroApiEndpoint=https://maestro-prod.westus2.cloudapp.azure.com
-            /p:Configuration=$(_BuildConfig)
+        - task: PowerShell@2
```

Comment:
is this change intentional? looks like this is an arcade file

Reply:
I just merged against master. Is this different from your copy of publish-build-assets.yml ?

Reply:
I windiff'd with master - they are identical after this merge.